tests/app/dao/test_ft_billing_dao.py

```diff
@@ -883,6 +883,7 @@ def test_fetch_letter_line_items_for_all_service(notify_db_session):
 def test_fetch_usage_year_for_organisation(notify_db_session):
     fixtures = set_up_usage_data(datetime(2019, 5, 1))
     service_with_emails_for_org = create_service(service_name='Service with emails for org')
+    create_annual_billing(service_with_emails_for_org.id, free_sms_fragment_limit=0, financial_year_start=2019)
```

Comment:
Optional: Did you consider whenever we call `create_service` it also creates a row in the annual billing table automatically so we can't forget to do it? 

Note, may not be a useful idea even if you haven't considered it

Reply:
Decided not to bother - as it'll affect a lot of tests that don't need annual billing, and also, it isn't clear what the value should be, especially for `financial_year_start` as that depends on what the test is looking at.